Makefile

```diff
@@ -1,6 +1,5 @@
 
 DOCKER ?= $(shell command -v docker)
-PANDOC ?= $(shell command -v pandoc)
```

Comment:
and also making the path to the input and output files

On Mon, May 23, 2016 at 3:31 PM W. Trevor King <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/opencontainers/runtime-spec/pull/440#discussion_r64273634>
> :
>
> > @@ -1,6 +1,5 @@
> >
> >  DOCKER ?= $(shell command -v docker)
> > -PANDOC ?= $(shell command -v pandoc)
>
> On Mon, May 23, 2016 at 12:23:18PM -0700, Vincent Batts wrote: Otherwise
> the whole command for the `docs` target will have to be overhauled to allow
> for a local pandoc binary.
> How difficult would this be? It looks like it's just moving ‘$(DOCKER) …
> -u $(shell id -u)’ into a PANDOC fallback (if the shell can find a local
> ‘docker’ but not a local ‘pandoc’).
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/runtime-spec/pull/440/files/5c27e4256d36759e7d1055e9a97e931a1cf79124#r64273634>
>